### PR TITLE
refactor GetWork to riteway

### PR DESCRIPTION
**Pull request by krobi64**
[PR Process](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-process) - [PR Review Checklist](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-review-checklist)

### Release
 Semantic release is enabled for this repository. Make sure you follow the right commit message convention. 
 We're using semantic-release's default — [Angular Commit Message Conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines).

## Description of Changes

References #585 

refactor alsatian GetWork to `riteway`

### CODE REVIEW

File: tests/integration/API/GetWork.test.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  // Submit a claim
+  await postWorkToNode(TheRaven)
+
+  {
+    const theRavenResponse = await getWorkFromNode(TheRaven.id)
+    const theRavenClaim = await theRavenResponse.json()
+
+    assert({
+      given: 'an id of a submitted work',
+      should: 'reuturn ok',
```
Comment (warrenv) [inline tests/integration/API/GetWork.test.ts]: spelling.

**Comment by kennylavender** [comment]
@warrenv this one should be in the functional folder shouldn't it? I understood the previous one being an integration test as it was testing secuirty headers, but this one is testing a user use case of the API.

**Comment by kennylavender** [comment]
@krobi64 don't await an answer to my question, feel free to merge, its simple enough to move the file if needed.

**Comment by warrenv** [comment]
@kennylavender I think this is an integration test. It is testing the route, controller and database. It is not testing that the claim was written to the blockchain, messages posted to queues, bitcoind sync, etc. The latter is handled by the functional tests. This integration test is focused on a subset of the use case, includes error cases and runs faster than a functional test.

**Pull request merged**